app/experimenter/experiments/tests/test_models.py

```diff
@@ -101,29 +101,46 @@ def test_bugzilla_url_returns_url_when_bugzilla_id_is_set(self):
             settings.BUGZILLA_DETAIL_URL.format(id=experiment.bugzilla_id),
         )
 
-    def test_test_tube_url_is_none_when_experiment_not_begun(self):
+    def test_monitoring_dashboard_url_is_none_when_experiment_not_begun(self):
         experiment = ExperimentFactory.create(
-            slug="experiment", status=Experiment.STATUS_DRAFT
+            slug="experiment",
+            status=Experiment.STATUS_DRAFT,
+            normandy_slug="normandy-slug",
         )
-        self.assertIsNone(experiment.test_tube_url)
+        self.assertIsNone(experiment.monitoring_dashboard_url)
 
-    def test_test_tube_url_is_none_when_experiment_is_addon_and_begun(self):
+    def test_monitoring_dashboard_url_returns_url_when_experiment_is_begun(
+        self
+    ):
         experiment = ExperimentFactory.create(
-            type=Experiment.TYPE_ADDON,
+            type=Experiment.TYPE_PREF,
             slug="experiment",
             status=Experiment.STATUS_LIVE,
+            normandy_slug="normandy-slug",
+        )
+        self.assertEqual(
+            experiment.monitoring_dashboard_url,
+            (
+                "https://grafana.telemetry.mozilla.org/d/3QA87kliz/"
+                "experiment-enrollment?orgId=1&var-experiment_id=normandy-slug"
```

Comment:
Again, oddly specific and also repeated. And I see it repeated one more time further down in this same file. 

If you change `models.py` to:
```python
    @property
    def monitoring_dashboard_url(self):
        if self.is_begun and self.normandy_slug:
            return settings.TELEMETRY_URL_TEMPLATE.format(slug=self.normandy_slug)
```

then, in `settings_test.py` you can override the default to something like:

```python
TELEMETRY_URL_TEMPLATE = "https://telemetry.example.com/?var-experiment_id={slug}"
```

And additionally, the tests could/should not be hard coded strings either and should probably be using...

```python
# UNTESTED
self.assertEqual(experiment.monitoring_dashboard_url,
  self.settings.TELEMETRY_URL_TEMPLATE.format('normandy-slug'))
```